backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -71,23 +79,23 @@ public Organization getCurrentOrganization() {
     public Organization getOrganization(String externalId) {
         Optional<Organization> found = _repo.findByExternalId(externalId);
         if (found.isEmpty()) {
-            throw new IllegalGraphqlArgumentException("Organization could not be found");
-        } else {
-            return found.get();
-        }
+            throw new IllegalGraphqlArgumentException("An organization with that external ID does not exist");
+        } 
+        
+        return found.get();
     }
 
     @AuthorizationConfiguration.RequireGlobalAdminUser
     public List<Organization> getOrganizations() {
         return _repo.findAll();
     }
 
-    public Organization getOrganizationForUser(ApiUser apiUser) {
+    public Optional<Organization> getOrganizationForUser(ApiUser apiUser) {
         String orgExternalId = _oktaService.getOrganizationExternalIdForUser(apiUser.getLoginEmail());
         if (orgExternalId == null) {
-            return null;
+            return Optional.empty();
         }
-        return getOrganization(orgExternalId);
+        return Optional.of(getOrganization(orgExternalId));
```

Comment:
ofNullable? I guess getOrganization throws an exception instead of returning null, though.